bundles/org.openhab.binding.wemo/src/main/java/org/openhab/binding/wemo/internal/handler/WemoHandler.java

```diff
@@ -110,63 +102,100 @@ public WemoHandler(Thing thing, UpnpIOService upnpIOService, WemoHttpCall wemoHt
     @Override
     public void initialize() {
         Configuration configuration = getConfig();
+        String udn = (String) configuration.get(UDN);
 
-        if (configuration.get("udn") != null) {
-            logger.debug("Initializing WemoHandler for UDN '{}'", configuration.get("udn"));
+        if (udn != null && !udn.isEmpty()) {
+            logger.debug("Initializing WemoHandler for UDN '{}'", udn);
             service.registerParticipant(this);
-            onSubscription();
-            onUpdate();
+
+            pollingJob = scheduler.scheduleWithFixedDelay(this::poll, 0, DEFAULT_REFRESH_INTERVALL_SECONDS,
+                    TimeUnit.SECONDS);
             updateStatus(ThingStatus.ONLINE);
         } else {
+            updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_ERROR,
```

Comment:
Should be ONLINE - see https://www.openhab.org/docs/concepts/things.html#status-details. Same for all handlers.

Reply:
NO, the link clearly says OFFLINE for Configuration_ERROR